exercises/change/change_test.exs

```diff
@@ -8,43 +8,70 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule ChangeTest do
   use ExUnit.Case
 
-  test "returns :error on empty list" do
-    assert Change.generate(1, []) == :error
+  # @tag :pending
+  test "no coins make 0 change" do
+      coins = [1, 5, 10, 21, 25]
```

Comment:
Also, as long as we are updating to match the canonical examples we should confirm to their order as well.